.env.template

```
@@ -7,3 +7,7 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
+
+TRELLO_API_KEY=XXXXX
+TRELLO_API_TOKEN=XXXXX
```

I think a small mismatch in variable names between this and your code; not an important one so fine to change here or in your code!
```suggestion
TRELLO_TOKEN=XXXXX
```